onnxt5/api.py

```diff
@@ -1,34 +1,41 @@
-import os
+from pathlib import Path
 import tarfile
+
+import boto3
 from onnxruntime import InferenceSession
+from tqdm import tqdm
 from transformers import T5Tokenizer
-import requests
 
-import onnxt5
+_package_path = Path(__file__).resolve().parent
+_models_path = _package_path.joinpath('model_data')
 
 
 def get_encoder_decoder_tokenizer():
     """ Function to get a default pre-trained version of T5 in ONNX ready for use """
-    package_path = os.path.dirname(onnxt5.__file__)
-    path_t5_encoder = os.path.join(package_path, 'model_data', 't5-encoder.onnx')
-    path_t5_decoder = os.path.join(package_path, 'model_data', 't5-decoder-with-lm-head.onnx')
+    path_t5_encoder = _models_path.joinpath('t5-encoder.onnx')
+    path_t5_decoder = _models_path.joinpath('t5-decoder-with-lm-head.onnx')
+
+    _models_path.mkdir(exist_ok=True)
```

Comment:
This library failed for me when running locally because the model data directory didn't exist